ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/OptimizeService.java

```diff
@@ -271,31 +290,22 @@ private void loadTables() {
     }
   }
 
-  private TableMetadata buildTableMetadata(Map<String, ArcticCatalog> icebergCatalogMap,
-                                           TableIdentifier tableIdentifier) {
+  private TableMetadata buildTableMetadata(ArcticCatalog arcticCatalog, ArcticTable arcticTable) {
```

Comment:
`buildTableMetadata(ArcticCatalog arcticCatalog, TableIdentifier tableIdentifier)` may be better, since if a table is not from the iceberg catalog, we don't need to load and get the ArcticTable to get the `TableMetadata`.